invenio_rdm_records/services/tasks.py

```diff
@@ -8,12 +8,30 @@
 """Celery tasks."""
 
 from celery import shared_task
+from datetime import timedelta, datetime
 from flask import current_app
+import math
 from invenio_access.permissions import system_identity
+from invenio_search.engine import dsl, search
+from invenio_search.proxies import current_search_client
+from invenio_search.utils import prefix_index
+
+from invenio_stats.aggregations import BookmarkAPI
```

Comment:
Reusing this class... I wonder if it would be better to put it in invenio_stats.bookmarks